src/Command/LintTemplatesCommand.php

```diff
@@ -509,6 +496,33 @@ private function assertValidBooleanProperty(string $property, array $attribute,
         }
     }
 
+    private function assertValidAttributeValidationRule(array $attribute, int $index, string $fileName, array $violations): void
+    {
+        $propertyPath = sprintf('[attributes][%d][validation_rule]', $index);
+        switch (true) {
+            case !array_key_exists('validation_rule', $attribute):
+                $violations[$fileName]->add(new ConstraintViolation(
+                    'This field is missing.',
+                    null,
+                    [],
+                    null,
+                    $propertyPath,
+                    null,
+                ));
+                break;
+            case !empty($attribute['validation_rule']) && !in_array($attribute['validation_rule'], self::VALIDATION_RULES):
```

Comment:
empty will validate ""
```suggestion
            case null !== ($attribute['validation_rule']) !in_array($attribute['validation_rule'], self::VALIDATION_RULES):
```